docs/notebooks/introduction.ipynb

```diff
@@ -849,6 +823,31 @@
         "\n",
         "\n"
       ]
+    },
+    {
+      "cell_type": "code",
+      "source": [],
+      "metadata": {
+        "id": "T8Pz8RXQJ3-9"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Performance\n",
+        "\n",
+        "CASA is now running performance benchmarks against a subset of the `casatasks` API to track various runtime metrics over development history of the project, starting in CASA 6. See the [Performance Benchmark](https://casangi.github.io/casabench/) webpages for an interactive view of the latest test results.\n",
+        "\n",
+        "### casabench\n",
+        "\n",
+        "The tests are implemented using [airspeed-velocity](https://asv.readthedocs.io/en/stable/) and tracked by a separate dedicated [repository](https://github.com/casangi/casabench). Automated deployment is coordinated with Bamboo, confined to a single computer node, and currently runs for each CASA6 pre-release package once other verification steps are complete. New tests and test cases will be added as development continues in CASA.\n",
```

Comment:
On this line, can you please expand it to give the available RAM of the system, ncores and type of processor? Thanks

Reply:
I added this information, plus also some details about the current OS